integration/spark/README.md

```diff
@@ -72,61 +72,72 @@ spark = (SparkSession.builder.master('local').appName('rdd_to_dataframe')
              .config('spark.jars', file)
              .config('spark.jars.packages', 'org.postgresql:postgresql:42.2.+')
              .config('spark.extraListeners', 'io.openlineage.spark.agent.OpenLineageSparkListener')
-             .config('spark.openlineage.url', 'http://{openlineage.client.host}/api/v1/namespaces/spark_integration/')
+             .config('spark.openlineage.transport.type', 'http')
+             .config('spark.openlineage.transport.url', 'http://{openlineage.client.host}/api/v1/namespaces/spark_integration/')
              .getOrCreate())
 ```
 
 ## Arguments
 
 ### Spark Listener
 The SparkListener reads its configuration from SparkConf parameters. These can be specified on the
-command line or in the `conf/spark-defaults.conf` file. 
+command line or in the `conf/spark-defaults.conf` file.
 
-There are two ways of supplying parameters, each parameter as separate config entry or all the parameters in 
-`spark.openlineage.url`
+The following parameters can be specified in the Spark configuration:
 
-NOTE: If `spark.openlineage.url` is defined, all the parameters will be taken from it regardless of whether they are
-also defined in configuration or not.
+### General
 
-#### Config entries
+Parameters configuring the Spark integration
 
-The following parameters can be specified in the Spark configuration:
+| Parameter                                | Definition                                                                                          | Example                         |
```

Comment:
Please provide some comment in PR on what has changed in README.

Reply:
I've added something